hail/src/main/scala/is/hail/backend/service/ServiceBackend.scala

```diff
@@ -433,7 +433,7 @@ object ServiceBackendSocketAPI2 {
       HailContext.get.backend = backend
       backend.addDefaultReferences()
     } else {
-      HailContext(backend, "hail.log", false, false, 50, skipLoggingConfiguration = true, 3)
+      HailContext(backend, 50, 3)
```

Comment:
did you confirm that service backend logs still appear?

Reply:
testing now, but I don't think I changed that since we were passing skip=True before

Reply:
(pretty sure for service we just pull in the log4j.properties file)

Reply:
logging works fine in QoB